examples/common_vmseries/.header.md

```diff
@@ -16,7 +16,7 @@ common VM-Series for all traffic; for a discussion of other options, please see
 
 ## Reference Architecture Design
 
-![simple](https://github.com/PaloAltoNetworks/terraform-azurerm-vmseries-modules/assets/6574404/a7c2452d-f926-49da-bf21-9d840282a0a2)
+![simple](https://github.com/PaloAltoNetworks/terraform-azurerm-swfw-modules/assets/2110772/aa2ae33a-fb46-4a1c-9811-98ea3b132297)
```

Comment:
I believe this link is auto generated when we uploaded the files to github?

Reply:
I'm not sure, took it from Sebastian's PR so it doesn't clash. I checked that the links are working.